src/app/pages/pages.routing.ts

```diff
@@ -1,10 +1,10 @@
 import { Routes, RouterModule } from '@angular/router';
 import { NgModule } from '@angular/core';
 
-
-
 import { HomeComponent } from './home/home.component';
 import { PagesComponent } from './pages.component';
+import { EventoComponent } from './evento/evento.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7138940)
```suggestion
import { EventoComponent } from "./evento/evento.component";
```